falcon/request.py

```diff
@@ -400,11 +402,25 @@ def range(self):
         except ValueError:
```

Comment:
L397 still references "bytes" (GH won't let me comment on the line directly since it wasn't modified in the PR).